src/main/kotlin/com/cognifide/gradle/aem/common/instance/LocalInstance.kt

```diff
@@ -116,6 +116,10 @@ class LocalInstance private constructor(aem: AemExtension) : AbstractInstance(ae
     val statusScript: Script
         get() = binScript("status")
 
+    @get:JsonIgnore
+    val touched: Boolean
```

Comment:
this is soft marker that there are some files to be deleted related with instance